docs/preview/features/secret-store/provider/hashicorp-vault.md

```diff
@@ -95,14 +95,21 @@ public class Program
                             secretPath: "my-secrets");
 
                         // Version of the KeyValue secret engine (default: V2).
-                         builder.AddHashiCorpVault(..., keyValueVersion: VaultKeyValueSecretEngineVersion.V1);
+                         builder.AddHashiCorpVault(..., options => options.KeyValueVersion: VaultKeyValueSecretEngineVersion.V1);
```

Comment:
Was this documentation outdated or breaking change?

Reply:
If this was not shipped yet, then it's ok

Reply:
Not only the documentation was changed, the extension and signature too. And yes, this isn't shipped yet in a minor version (ony previews) so I guess it's OK to still break this.